lexrpc/client.py

```diff
@@ -169,10 +173,22 @@ def call(self, nsid, input=None, **params):
         if type == 'subscription':
             return self._subscribe(url)
 
+        def is_json(x):
```

Comment:
hah, nice!...but fortunately we can probably simplify to just `isinstance(..., dict)` since XRPC JSON inputs/outputs are always objects, https://atproto.com/specs/lexicon#query-and-procedure-http-api , and/or check against the lexicon's input encoding.